### DISCUSSION TITLE
第5节课作业(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#423] [作业]
作业提交

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection5-LLM-Quantization-Deployment-Practice-based-on-LMDeploy/

**Comment by PlateAss** [comment]
https://plateass.github.io/work5

**Comment by yaoyaotaba** [comment]
https://zhuanlan.zhihu.com/p/679007570

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135765410

**Comment by HuJijin** [comment]
https://polished-collision-b6f.notion.site/92a390bbda42452898486d41008e7b12?pvs=4

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/0xcEe
